spec/factories/department.rb

```diff
@@ -0,0 +1,5 @@
+FactoryBot.define do
+  factory :department, class: Feedbook::Department do
+    name 'Department'
+  end
+end
```

Comment:
Github complains that there is no newline at the end of the file, maybe your editor is not using LF notation.
What do you think about to add a newline at end of the file?